### PR TITLE
Fixes: only replaces first instance of invalid character

### LINKED ISSUE #30: only replaces first instance of invalid character
The example given in the README fails...

```js
var cleanGitRef = require('clean-git-ref');
assert.stricEqual(cleanGitRef.clean('bad git ref formats/'), 'bad-git-ref-formats');
```

it turns `'bad git ref formats/'` into `'bad-git ref formats'`.

### PR DESCRIPTION
**Pull request by thvd**
Also this fixes #30

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,21 +1,35 @@
 'use strict';
 
+
```
Comment (elicwhite) [inline src/index.js]: Looks like this is causing an eslint error, failing the Travis build

**Review by elicwhite — COMMENTED** [review]
Thanks for the PR! One nit and then I can merge it.

File: src/index.js (hunk 1 above)
Comment (thvd) [reply src/index.js]: Yes thanks, got it fixed in 81a8287

**Comment by elicwhite** [comment]
Published as 2.0.1

**Pull request merged**